playbooks/aws/openshift-cluster/vars.yml

```diff
@@ -1,26 +1,29 @@
 ---
 
-clusterid: mycluster
+clusterid: opstest
 region: us-east-1
 
 provision:
   clusterid: "{{ clusterid }}"
   region: "{{ region }}"
 
+  create_vpc: True
```

Comment:
This will not allow users to override with inventory/group_vars.

Reply:
This is only for this playbook.  The parameter in the role is `openshift_aws_create_vpc` and can be overridden as well as the other values.  Same question as before, how do you propose setting values for this playbook?

I could create a `vars.example.yaml` and then expect the user to use their own version.